### ISSUE TITLE
Create an UI for user profiles.

### ISSUE THREAD

**Issue by yashnerkar** [CSI-CATT-DMCE/csi_badge#14]
UI for searching an User and display the user.
Reference : [Search Bar](https://github.com/CSI-CATT-DMCE/csi_badge/blob/main/Screenshots/user%20profiles/Search%20User.png)
[Display profile](https://github.com/CSI-CATT-DMCE/csi_badge/blob/main/Screenshots/user%20profiles/User%20Profile.png)

**Comment by Akku2016** [comment]
which language we can make it can you tell me plz

**Comment by yash-png** [comment]
You have to use HTML, CSS and for responsive use the  latest version of bootstrap

**Comment by Akku2016** [comment]
can i take this one

**Comment by yash-png** [comment]
yeah! but you have to make two separate pages. One is for the search bar and another is the user's profile.

**Comment by Akku2016** [comment]
ok i will try .plz assign to me

**Comment by yash-png** [comment]
okay go ahead then

**Comment by Akku2016** [comment]
in user profile you mentioned two image 👎 
like this i have to designed for both pages 
tell me if i am correct or not

**Comment by yash-png** [comment]
yes

**Comment by Akku2016** [comment]
hi, I am beginner in bootstrap that why I took time.
 I did  first design. plz check out if this is wrong plz correct me and then I can start another design:-
https://user-images.githubusercontent.com/88783468/143616756-2b823937-3472-4e9c-b0f7-1a5efa67a635.PNG

**Comment by yash-png** [comment]
The only thing is to stick the navbar to the top. don't put a gap in it

**Comment by Akku2016** [comment]
ok I will correct that one. In user profile I have one  doubt 
where I have to put user profile icon and
 one more thing I have to put in down  "scroll bar " for seeing the second row box .plz correct me if I am wrong

**Comment by yash-png** [comment]
yes you have to add scroll bar..
you can put user profile icon in the center of div itself

**Comment by Karan2310** [comment]
Can I work on it?

**Comment by CSI-CATT-DMCE** [comment]
yeah go ahead, check for the references as well
